test/snippet/search/Algorithm/search.cpp

```diff
@@ -0,0 +1,21 @@
+#include <vector>
+
+#include <seqan3/alphabet/nucleotide/dna4.hpp>
+#include <seqan3/core/debug_stream.hpp>
+#include <seqan3/search/algorithm/search.hpp>
+#include <seqan3/search/fm_index/all.hpp>
+
+int main()
+{
+    using seqan3::operator""_dna4;
+    std::vector<std::vector<seqan3::dna4>> genomes{"CGCTGTCTGAAGGATGAGTGTCAGCCAGTGTA"_dna4,
+                                                   "ACCCGATGAGCTACCCAGTAGTCGAACTG"_dna4,
+                                                   "GGCCAGACAACCCGGCGCTAATGCACTCA"_dna4};
+    std::vector<std::vector<seqan3::dna4>> queries{"GCT"_dna4, "ACCC"_dna4};
+
+    seqan3::bi_fm_index index{genomes};                                // build the index
+    
+    // search for the queries "GCT" and "ACCC"
+    seqan3::debug_stream << seqan3::search(queries, index) << '\n'; // [[(0,1),(1,9),(2,16)],[(1,0),(1,12),(2,9)]]
```

Comment:
Can you instead of a single comment `// [[(0,1),(1,9),(2,16)],[(1,0),(1,12),(2,9)]]` 
explain the output a bit? e.g.
```cpp
// This prints the result vector [[(0,1),(1,9),(2,16)],[(1,0),(1,12),(2,9)]]
// which is ... 
```